support-frontend/assets/pages/paper-subscription-checkout/components/appsSection.tsx

```diff
@@ -45,7 +46,7 @@ const mainHeading = css`
 	${from.phablet} {
 		${headline.medium({
 			fontWeight: 'bold',
-			lineHeight: 'normal',
+			lineHeight: 'regular',
```

Comment:
The options listed in the lineHeight type over in Source are `"tight" | "regular" | "loose"`. 'Regular' seemed the right equivalent to 'normal'.